strings/alternative_string_arrange.py

```diff
@@ -0,0 +1,31 @@
+def alternative_string_arrange(first_str: str, second_str: str) -> str:
+    """
+    Return the alternative arrangements of the two strings.
+    :param first_str:
+    :param second_str:
+    :return: String
+    >>> alternative_string_arrange("ABCD", "XY")
+    'AXBYCD'
+    >>> alternative_string_arrange("XY", "ABCD")
+    'XAYBCD'
+    >>> alternative_string_arrange("AB", "XYZ")
+    'AXBYZ'
+    >>> alternative_string_arrange("ABC", "")
+    'ABC'
+    """
+    first_str_length: int = len(first_str)
+    second_str_length: int = len(second_str)
+    abs_length: int = (
+        first_str_length if first_str_length > second_str_length else second_str_length
+    )
+    li: list = []
```

Comment:
I don't like name `li`. If you name it for example `output_list` would be more readable.

Reply:
Sure, I change it.